README.md

```diff
@@ -79,87 +79,146 @@ make helm-chart-push
 ```
 
 
-## Deploy Fybrik module
-1. In your module yaml spec (`hello-world-read-module.yaml`):
-    - Change `spec.chart.name` to your chart registry.
-    - Define `flows` and `capabilities` for your module, an example can be found in `hello-world-read-module.yaml`. 
+## Register as a Fybrik module
+
+To register HWRM (Hello Read World Module) as a Fybrik module apply `hello-world-read-module.yaml` to the fybrik-system namespace of your cluster.
+
+To install the latest release run:
 
-2. Deploy `FybrikModule` in `fybrik-system` namespace:
 ```bash
-kubectl create -f hello-world-read-module.yaml -n fybrik-system
+kubectl apply -f https://github.com/fybrik/hello-world-read-module/releases/latest/download/hello-world-read-module.yaml -n fybrik-system
 ```
-3. Check if `FybrikApplication` successfully deployed:
+
+### Version compatbility matrix
+
+| Fybrik           | HWRM     | Command
+| ---              | ---     | ---
+| 0.5.x            | 0.5.x   | `https://github.com/fybrik/hello-world-read-module/releases/download/v0.5.0/hello-world-read-module.yaml`
+| master           | main    | `https://raw.githubusercontent.com/fybrik/hello-world-read-module/main/hello-world-read-module.yaml`
+
+
+## Deploy and test Fybrik module
+
+Follow this section to deploy and test the module on a single cluster.
+
+### Before you begin
+
+Install Fybrik using the [Quick Start](https://fybrik.io/v0.5/get-started/quickstart/) guide. This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
+
+> ***Notice: Please follow `version compatbility matrix` section above for deploying the correct version of Fybrik and this module.***
+
+### Deploy Fybrik module
+
+Deploy `FybrikModule` in `fybrik-system` namespace:
 ```bash
-kubectl get fybrikmodule hello-world-read-module -n fybrik-system
-kubectl describe fybrikmodule hello-world-read-module -n fybrik-system
+kubectl create -f hello-world-read-module.yaml -n fybrik-system
```

Comment:
P.S. `kubectl create` will fail if the module exists

Reply:
Changed to `apply`